### DISCUSSION TITLE
Yolov8 .pt to ONNX model

### DISCUSSION THREAD

**Discussion by k-Rohit** [ultralytics/ultralytics#3387] [Q&A]
I want to know how we can convert yolov8.pt to ONNX format and use it just like .pt model

**Comment by glenn-jocher** [comment]
@k-Rohit hello there! Converting a YOLOv8 `.pt` model to ONNX format is a straightforward process using the `export` mode provided by Ultralytics YOLOv8. Here's a brief overview of the steps:

1. Ensure you have the latest version of the Ultralytics YOLOv8 repository and all the required dependencies installed.
2. Use the `export` mode with the appropriate command-line arguments to specify the input `.pt` file and the desired output ONNX format.
3. Once the export is complete, you'll have an ONNX model that you can use in environments that support ONNX.

For detailed instructions and options available during the export process, please refer to the Export section in our documentation at https://docs.ultralytics.com/modes/export. This will guide you through the necessary steps and provide additional information on how to work with the exported ONNX model.

If you encounter any issues or have further questions, feel free to reach out. Happy coding! 😊🚀
